cluster/gce/manifests/konnectivity-server.yaml

```diff
@@ -7,7 +7,8 @@ metadata:
     seccomp.security.alpha.kubernetes.io/pod: 'docker/default'
   component: konnectivity-server
 spec:
-  priorityClassName: system-cluster-critical
+  priorityClassName: system-node-critical
```

Comment:
cc @caesarxuchao, since this is using UDS, it needs to run on this node (can't be rescheduled to another node)